src/pydocstyle/checker.py

```diff
@@ -284,10 +284,12 @@ def check_indent(self, definition, docstring):
                 indents = [leading_space(l) for l in lines if not is_blank(l)]
                 if set(' \t') == set(''.join(indents) + indent):
                     yield violations.D206()
-                if (len(indents) > 1 and min(indents[:-1]) > indent or
-                        indents[-1] > indent):
+                if (
+                    len(indents) > 1 and min(indents[:-1]) > indent or
```

Comment:
Probably worth putting brackets for clarity here?

Reply:
there weren't brackets before, the newline I think is enough to differentiate, I can change it if you insist (or you can push to the PR if you like)